model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -248,3 +265,6 @@ private class ResolvedLicenseBuilder(val license: SpdxSingleLicenseExpression) {
 
     fun build() = ResolvedLicense(license, originalDeclaredLicenses, originalExpressions, locations)
 }
+
+private val UNDEFINED_TEXT_LOCATION = TextLocation("/", TextLocation.UNKNOWN_LINE, TextLocation.UNKNOWN_LINE)
```

Comment:
Consider using "." instead of "/" which at least does not point to the file system root if something goes wrong 😉